examples/window/window_settings.rs

```diff
@@ -46,16 +46,20 @@ fn main() {
                 toggle_window_controls,
                 cycle_cursor_icon,
                 switch_level,
+                toggle_visible,
             ),
         )
         .run();
 }
 
-fn setup(mut primary_window: Query<&mut Window, With<PrimaryWindow>>) {
-    // At this point the gpu is ready to show the app so we can make the window visible
-    // There might still be a white frame when doing it in startup.
-    // Alternatively, you could have a system that waits a few seconds before making the window visible.
-    primary_window.single_mut().visible = true;
+fn toggle_visible(mut window: Query<&mut Window>, frames: Res<FrameCount>) {
```

Comment:
I don't have a good suggestion, but other `toggle_` systems in this example are triggered by user input, while this one is more like a delayed setup

Reply:
Good point, how about `make_visible` or `automatically_toggle_visible` to make it more obvious?